app/src/main/java/org/sagebionetworks/research/mpower/MPowerRadioButton.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.research.mpower;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Paint;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.constraint.ConstraintLayout;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MPowerRadioButton extends ConstraintLayout {
```

Comment:
It's been awhile since I've made a custom RadioButton.  But for any button, the drawables can be a bit complicated, because they require extra info about what drawable to use depending on state.  It sometimes gets even more complicated with the the re-design in api 21, but I'm not sure if that is applicable to RadioButton.  Have you tried the solution here? https://stackoverflow.com/questions/12432553/radiobutton-how-to-use-a-custom-drawable

Reply:
I have tried this, the problem is not the drawable not being in the correct state, it is the drawable not appearing at all.